### PR TITLE
Adds recitation 1

### PR DESCRIPTION
**Pull request by anudaweerasinghe**
Adds [handout](https://docs.google.com/document/d/1Cxl6L3SZ4tLXxC3Wvu1crE1Z6gZKng5OIke4zbjdiTw/edit#)

### CODE REVIEW

**Comment by victorhuangwq** [comment]
@angelaz1 the netlify page seems to show that the recitation as still MLK? Is there a way to update that?

**Comment by angelaz1** [comment]
> @angelaz1 the netlify page seems to show that the recitation as still MLK? Is there a way to update that?

The “this week” updates on Sunday midnight - you can also manually run the script  `docs/_data/update_this_week.py`

**Comment by angelaz1** [comment]
Wait sorry I just got a chance to look at this
Is this the normal GitHub recitation? I converted this into the GitHub exercise with hw1, wouldn’t this be a duplicate?

File: docs/recitations/reci1-github.md [hunk 1]
```diff
@@ -5,64 +5,73 @@ title: Recitation 1 - Git Collaboration
...
+
+- Have either you or your partner fork [this repository](https://github.com/CMU-313/github-recitation-ts), then add the other person as a collaborator on GitHub. You can do so by clicking on **Settings**, then **Collaborators**
 - Under the General tab in **Settings**, check the "Issues" checkbox in the "Features" section to enable the issues pane for your repository
-![Github Permissions](../assets/images/reci/reci1-github-permissions.png)
+  ![Github Permissions](../assets/images/reci/reci1-github-permissions.png)
 - Clone the repository onto your local machine
 
 ## Activity 1b: Run a GitHub Action Workflow
+
 - On Github, click the **Actions** tab and select the "Python Application" workflow
```
Comment (angelaz1) [inline docs/recitations/reci1-github.md:29]: Looks like the workflow is called “Lint and Test”, not Python Application

**Review by angelaz1 — CHANGES_REQUESTED** [review]
Small change in handout + need to add the recitation to `docs/_data/schedule.yml` so it shows up in the schedule. You may also want to check against the Hw1 github exercise to make sure we aren’t being redundant with some of the info; o/w looks ok

**Review by angelaz1 — APPROVED** [review]
Look good! Made some quick edits to the schedule file so "This Week" shows up nicely

**Pull request merged**